src/librbd/crypto/luks/FormatLUKSRequest.h

```diff
@@ -0,0 +1,51 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef CEPH_LIBRBD_CRYPTO_LUKS_FORMAT_LUKS_REQUEST_H
```

Comment:
Nit: since we are in the `luks` namespace, we can probably safely drop the extra `LUKS` from the class names / include guards